cli/generator-op-microservice/generators/microservice/templates/src/e2e/rest.service.spec.ts

```diff
@@ -0,0 +1,85 @@
+import dotenv from 'dotenv';
+/* Mock */
+import mock from './mock.json';
+import path from 'path';
+import supertest from 'supertest';
+
+dotenv.config({ path: path.resolve(__dirname, '.test.env')});
```

Comment:
This is already configured in the webpack dotenv plugin. No need to configure it here again.